hr_course/models/hr_course.py

```diff
@@ -0,0 +1,213 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class HrCourse(models.Model):
+    _name = "hr.course"
+    _description = "Course"
+    _inherit = "mail.thread"
+
+    name = fields.Char(string="Name", required=True, tracking=True)
+    category_id = fields.Many2one(
+        "hr.course.category", string="Category", required=True,
+    )
+
+    start_date = fields.Date(
+        string="Start date",
+        readonly=True,
+        states={"draft": [("readonly", False)]},
+        tracking=True,
+    )
+    end_date = fields.Date(
+        string="End date",
+        readonly=True,
+        states={"draft": [("readonly", False)]},
+        tracking=True,
+    )
+    currency_id = fields.Many2one(
+        "res.currency",
+        string="Currency",
+        default=lambda self: self.env.user.company_id.currency_id,
+    )
+    cost = fields.Monetary(string="Course Cost", required=True, tracking=True)
+    authorized_by = fields.Many2one(
+        string="Authorized by",
+        comodel_name="hr.employee",
+        required=True,
+        readonly=True,
+        states={"draft": [("readonly", False)]},
+        tracking=True,
+    )
+    permanence = fields.Boolean(
+        string="Has Permanence",
+        readonly=True,
+        states={"draft": [("readonly", False)]},
+        tracking=True,
+    )
+    permanence_time = fields.Char(
```

Comment:
Not sure how to use this field ^^

Reply:
This is an information field used to tell if making this course implies that employees have to stay in the company for at least a certain amount of time.

Reply:
@jarroyomorales Hi, this field mean employee take at least `permanence time` in course. right?

Reply:
> This is an information field used to tell if making this course implies that employees have to stay in the company for at least a certain amount of time.

This was my idea, but maybe we need to rethink the usage of the field

Reply:
I think the fields are legit, but, a help would be recomended

Reply:
done!